src/components/TransactionPanel/TransactionPanel.js

```diff
@@ -213,6 +217,73 @@ export class TransactionPanelComponent extends Component {
     const isProviderBanned = isProviderLoaded && currentProvider.attributes.banned;
     const isProviderDeleted = isProviderLoaded && currentProvider.attributes.deleted;
 
+
+    // QR code %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+  const account = 'm';
+  const amount = 'a';
+  const product = 'r';
+  const category = 'cat';
+  const ven = 'ven';
+  const cbu = 'cbu';
+  const web = 'web';
+  const ind = 'ind';
+
+  const venObj = {
+    //[cbu]: canonicalRootURL + '/order/' + values.orderId.uuid + '/details',
+    [cbu]: canonicalRootURL + '/api/accept-privileged?txId=' + currentTransaction.id.uuid,
+    [web]: canonicalRootURL,
+    [ven]: 'Good Dollar Marketplace',
+    [ind]: 'used_for_what'
+  };
+
+
+  const obj = {
+    [account]: currentListing.author.attributes.profile.publicData.goodDollarAccount,
+    [amount]: currentListing.attributes.price.amount,
+    [product]: currentListing.attributes.title,
+    [category]: 'Other',
```

Comment:
dont we have categories in the marketplace?

Reply:
We understood from  [https://github.com/GoodDollar/GoodDAPP/blob/b5b4f253d1cccb35e5b37e9074592f8566ad3111/src/components/dashboard/Reason.js](url)  
and from [https://github.com/GoodDollar/GoodDAPP/blob/bcc3b98d34bf34ced20eca762425c96d1405281f/src/lib/share/index.js](url)
![WhatsApp Image 2021-08-29 at 09 27 00](https://user-images.githubusercontent.com/75620015/131240925-6a8e8180-7084-4c00-ab5e-4913fdfa4f7e.jpeg)

that a category is required. Also @emipasat , on his conversations with you on slack, understood that this is required. 

![WhatsApp Image 2021-08-29 at 09 19 08](https://user-images.githubusercontent.com/75620015/131240912-70f365e9-a89c-4a02-8ad1-9344673398d0.jpeg)

We can remove it if it's of no use.

Reply:
yes , we do. I will add the product's category there